### PR TITLE
Fix asr postprocessing nvbug

### PR DESCRIPTION
**Pull request by yzhang123**

### CODE REVIEW

File: examples/nlp/asr_postprocessor/asr_postprocessor.py [hunk 1]
```diff
@@ -60,7 +60,7 @@
 parser.add_argument("--attn_score_dropout", default=0.25, type=float)
 parser.add_argument("--attn_layer_dropout", default=0.25, type=float)
 parser.add_argument("--eval_step_frequency", default=2000, type=int)
-parser.add_argument("--data_dir", default="/dataset/", type=str)
+parser.add_argument("--data_dir", default="../../../tests/data/pred_real", type=str)
```
Comment (VahidooX) [inline examples/nlp/asr_postprocessor/asr_postprocessor.py]: Why are we using the data under test?

Comment (yzhang123) [reply examples/nlp/asr_postprocessor/asr_postprocessor.py]: it used to be there in our last release

Comment (VahidooX) [reply examples/nlp/asr_postprocessor/asr_postprocessor.py]: Where does this data come from?

Comment (VahidooX) [reply examples/nlp/asr_postprocessor/asr_postprocessor.py]: I can not find the data. Is it under here https://github.com/NVIDIA/NeMo/tree/master/tests/data?

Comment (yzhang123) [reply examples/nlp/asr_postprocessor/asr_postprocessor.py]: Yes

Comment (VahidooX) [reply examples/nlp/asr_postprocessor/asr_postprocessor.py]: In current master, there is no pred_real?

Comment (yzhang123) [reply examples/nlp/asr_postprocessor/asr_postprocessor.py]: No

Comment (yzhang123) [reply examples/nlp/asr_postprocessor/asr_postprocessor.py]: I added in this pr, please checkout the commits or changed files

Comment (VahidooX) [reply examples/nlp/asr_postprocessor/asr_postprocessor.py]: These files are just a small postion of the real data. I think these small files are created just for CI tests. You may merge this PR for now, but we should get the real data from Oleksii and share it with users.

**Pull request merged**